openedx/features/edly/tests/test_utils.py

```diff
@@ -25,13 +40,40 @@ def test_encode_edly_user_info_cookie(self):
         Test that "encode_edly_user_info_cookie" method encodes data correctly.
         """
         actual_encoded_string = encode_edly_user_info_cookie(self.test_edly_user_info_cookie_data)
-        expected_encoded_string = jwt.encode(self.test_edly_user_info_cookie_data, settings.EDLY_COOKIE_SECRET_KEY, algorithm=settings.EDLY_JWT_ALGORITHM)
+        expected_encoded_string = jwt.encode(
+            self.test_edly_user_info_cookie_data, settings.EDLY_COOKIE_SECRET_KEY,
+            algorithm=settings.EDLY_JWT_ALGORITHM
+        )
         assert actual_encoded_string == expected_encoded_string
 
     def test_decode_edly_user_info_cookie(self):
         """
         Test that "decode_edly_user_info_cookie" method decodes data correctly.
         """
-        encoded_data = jwt.encode(self.test_edly_user_info_cookie_data, settings.EDLY_COOKIE_SECRET_KEY, algorithm=settings.EDLY_JWT_ALGORITHM)
+        encoded_data = jwt.encode(
+            self.test_edly_user_info_cookie_data,
+            settings.EDLY_COOKIE_SECRET_KEY,
+            algorithm=settings.EDLY_JWT_ALGORITHM
+        )
         decoded_edly_user_info_cookie_data = decode_edly_user_info_cookie(encoded_data)
         assert self.test_edly_user_info_cookie_data == decoded_edly_user_info_cookie_data
+
+    def test_user_organization_access(self):
```

Comment:
match names of similar tests `test_user_with_organization_access`

Reply:
Done